### PR TITLE
Post-publish cleanup.

### PR DESCRIPTION
**Pull request by YevheniiSemendiak**

### CODE REVIEW

File: action.yaml [hunk 1]
```diff
@@ -18,18 +18,23 @@ inputs:
     descr: |
       Rclone WebDAV server port.
       Useful if you want to access the server within the cluster, for instance, from another job.
+  job_name:
+    default: "webdav"
+    descr: |
+      The name of WebDAV server.
+      Use it to generate a predictable job hostname.
```
Comment (asvetlov) [inline action.yaml:25]: Is the description still actual?

Comment (YevheniiSemendiak) [reply action.yaml:25]: Actually yes, we use this input to set the job name [here](https://github.com/neuro-actions/webdav_server/pull/1/files#diff-fab4d7fb461bc6fbe9587f6c03fff98102b1c744145edcf2a993f2ff7cb05a0dR33).
Or, maybe I've got your question wrong

Comment (asvetlov) [reply action.yaml:25]: I read it as "a string used to generate job name by concatenating with other things" but it is the whole job name actually.
Was `$[[ flow.title ]]-webdav` but now is `$[[ inputs.job_name ]]`.

**Comment by asvetlov** [comment]
Very minor: can you please use `${{ ... }}` notation everywhere except the cookie-cutter template maybe.
We constantly use curly-brackets notation in docs; there is no reason to confuse people by `$[[ ... ]]`

**Comment by YevheniiSemendiak** [comment]
Ok, I will replace squared brackets with curly ones for keeping the integrity with the docs.

File: action.yaml (hunk 1 above)
Comment (YevheniiSemendiak) [reply action.yaml:25]: True, initially it was a buggy `$[[ flow.title ]]-webdav` 🙂
I've replaced it with the proper input in 6c8e866

**Pull request merged**